examples/seismic/acoustic/accuracy.ipynb

```diff
@@ -286,86 +357,76 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
+   "execution_count": 12,
    "metadata": {},
    "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "geometry.time_axes;  TimeAxis: start=0, stop=150.08, step=0.08, num=1877\n"
+     ]
+    },
     {
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "GNUCompiler: compiled /var/folders/mx/qs0dn9rx7zn6dz2zvwv7tkk00000gn/T/devito-cjqed58a/7261032ccc57ff87dc245bd04b59bcf3be82b92b.c [0.33 s]\n",
-      "=========================================================================================\n",
-      "Section section_1<1874,1> with OI=0.62 computed in 0.002 s [0.06 GFlops/s]\n",
-      "Section section_2<1874,1> with OI=0.77 computed in 0.084 s [0.00 GFlops/s]\n",
-      "Section main<1874,861,861> with OI=2.86 computed in 8.354 s [11.97 GFlops/s, 0.17 GPts/s]\n",
-      "=========================================================================================\n"
+      "Operator `Forward` run in 20.87 s\n"
      ]
     },
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "error for dt= 0.08 is 0.000850230956588 \n",
-      "\n"
+      "error for dt=0.0800 is 0.001126 \n",
```

Comment:
Shouldn't we test for dt = 0.0875 ?

Reply:
dont follow ... 
`dt = [.1, 0.08, .075, 0.0625, .05]`

Reply:
If the step is -0.0125 should not it be 0.0875 ?

Reply:
I see, that would be more consistent, then will have. Might be a typo.
```
dt = [0.1000, 0.0875, .0750, 0.0625, .0500]
```

Reply:
done

Reply:
when I used 0.0875 I got a weird result, this may be why @mloubout  chose 0.08?

![image](https://user-images.githubusercontent.com/45494770/77827806-c3061980-70e5-11ea-8e15-a0e566a87074.png)

Reply:
Per @mloubout comment, 0.0875 not even divisible for 1500 ms, hence use of 0.0800.

Reply:
I did make the following change:
```
dt = [0.1000, 0.0800, 0.0750, 0.0625, 0.0500]
nnt = (np.divide(150.0, dt) + 1).astype(int)
```